tests/conftest.py

```diff
@@ -1,70 +1,253 @@
+    url = "https://github.com/MODFLOW-USGS/modflow6-nightly-build/releases/latest/download/"
+    sysinfo = platform.system()
+    if sysinfo == "Windows":
+        url += "win64.zip"
+        lib_path = tmp_path / "libmf6.dll"
+    elif sysinfo == "Linux":
+        url += "linux.zip"
+        lib_path = tmp_path / "libmf6.so"
+    elif sysinfo == "Darwin":
+        url += "mac.zip"
+        lib_path = tmp_path / "libmf6.so"
+
+    pymake.download_and_unzip(url=url, pth=str(tmp_path))
+    return str(lib_path)
+
+
+@dataclass
+class FlopyDis:
+    sim_path: str
+    sim: MFSimulation
+    tdis_rc: List[Tuple]
+    model_name: str
+    nrow: int
+    ncol: int
+    stress_period_data: List[Any]
+
+
+@pytest.fixture(scope="function")
+def flopy_dis(tmp_path, modflow_lib_path):
+    sim_path = str(tmp_path)
+    sim = flopy.mf6.MFSimulation(
+        sim_name="TEST_SIM_DIS", version="mf6", sim_ws=sim_path
+    )
+    flopy_dis = FlopyDis(
+        sim_path=sim_path,
+        sim=sim,
+        tdis_rc=[(6.0, 2, 1.0), (6.0, 3, 1.0)],
+        model_name="TEST_MODEL_DIS",
+        nrow=9,
+        ncol=10,
+        stress_period_data=[[(0, 2, 0), 1.0], [(0, 6, 8), 0.0]],
+    )
+    flopy.mf6.ModflowTdis(sim, time_units="DAYS", nper=2, perioddata=flopy_dis.tdis_rc)
+    flopy.mf6.ModflowIms(sim)
+    gwf = flopy.mf6.ModflowGwf(sim, modelname=flopy_dis.model_name, save_flows=True)
+    flopy.mf6.ModflowGwfdis(gwf, nrow=flopy_dis.nrow, ncol=flopy_dis.ncol)
+    flopy.mf6.ModflowGwfic(gwf)
+    flopy.mf6.ModflowGwfnpf(gwf, save_specific_discharge=True)
+    flopy.mf6.ModflowGwfchd(gwf, stress_period_data=flopy_dis.stress_period_data)
+    budget_file = flopy_dis.model_name + ".bud"
+    head_file = flopy_dis.model_name + ".hds"
+    flopy.mf6.ModflowGwfoc(
+        gwf,
+        budget_filerecord=budget_file,
+        head_filerecord=head_file,
+        saverecord=[("HEAD", "ALL"), ("BUDGET", "ALL")],
+    )
+    sim.write_simulation()
+    return flopy_dis
+
+
+@dataclass
+class FlopyDisu:
+    sim_path: str
+    sim: MFSimulation
+    tdis_rc: List[Tuple]
+    model_name: str
+    nlay: int
+    nrow: int
+    ncol: int
+
+
+@pytest.fixture(scope="function")
+def flopy_disu(tmp_path):
+    sim_path = str(tmp_path)
+    sim = flopy.mf6.MFSimulation(sim_name="TEST_SIM_DISU", sim_ws=sim_path)
+    flopy_disu = FlopyDisu(
+        sim_path=sim_path,
+        sim=sim,
+        tdis_rc=[(6.0, 2, 1.0), (6.0, 3, 1.0)],
+        model_name="TEST_MODEL_DISU",
+        nlay=1,
+        nrow=3,
+        ncol=3,
+    )
+    flopy.mf6.ModflowTdis(sim, time_units="DAYS", nper=2, perioddata=flopy_disu.tdis_rc)
+    flopy.mf6.ModflowIms(sim)
+    gwf = flopy.mf6.ModflowGwf(sim, modelname=flopy_disu.model_name, save_flows=True)
+    flopy.mf6.ModflowGwfdisu(
+        gwf,
+        nodes=9,
+        nja=33,
+        nvert=16,
+        top=[0.0],
+        bot=[-2.0],
+        area=np.full(9, 1.0),
+        iac=[3, 4, 3, 4, 5, 4, 3, 4, 3],
+        ja=[
+            0,

```

Comment:
Hi @Hofer-Julian, do you think we can do something about the auto-formatter such that the arrays are not spread over so many lines?